tests/network/vmi_multus.go

```diff
@@ -1407,25 +1410,29 @@ func getInterfaceNetworkNameByMAC(vmi *v1.VirtualMachineInstance, macAddress str
 }
 
 func validateSRIOVSetup(virtClient kubecli.KubevirtClient, sriovResourceName string, minRequiredNodes int) error {
-	nodes := util.GetAllSchedulableNodes(virtClient)
-	Expect(nodes.Items).ToNot(BeEmpty(), "There should be some compute node")
+	sriovNodes := getNodesWithAllocatedResource(virtClient, sriovResourceName)
+	if len(sriovNodes) < minRequiredNodes {
```

Comment:
nit: you could inline these, since IMO it reads well enough on its own.

```
if len(getNodesWithAllocatedResource(virtClient, sriovResourceName)) < minRequiredNodes {
...
}
```

Reply:
I wrote it like this at first then I got back to it and it seemed to me a bit bulky, I prefer to have that variable if you don't mind :slightly_smiling_face:

Reply:
Fair enough, I don't mind.